docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -356,7 +427,10 @@ Standard Kaldi\* Layers:
 | :----------| :----------|
 | addshift | No |
```

Comment:
Could you remove from Kaldi chapter Crop and softsign? We have no support for such operations from Kaldi framework.

Reply:
Done.